src/services/ApiService/Dex/Dex.ts

```diff
@@ -118,6 +118,7 @@ export default class DexService {
       data.allowanceTarget !== AddressZero
         ? {
             ...formatApproveTx({
+              fromAddress: (account as TAddress) ?? undefined,
```

Comment:
`?? undefined` seems redundant here?

Reply:
No, because `null` and `undefined` aren't the same

Reply:
`account` can be `string` or `null`

Reply:
Can't you change the type of account to `string | undefined` then?

Reply:
Sure